app/database/queries/breeze_buddy/telephony_number.py

```
@@ -180,34 +180,98 @@ def get_all_outbound_numbers_with_call_count_query(
+        if reseller_id is not None:
+            add("reseller_id", reseller_id)
+        if merchant_id is not None:
+            add("merchant_id", merchant_id)
+    sets.append('"updated_at" = NOW()')
+
+    text = f"""
+        UPDATE "{TELEPHONY_NUMBER_TABLE}"
+        SET {", ".join(sets)}
+        WHERE "id" = $1
+        RETURNING *;
+    """
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)